api/v1/requests/JobApplicationRequest.js

```diff
@@ -0,0 +1,20 @@
+const Request = require('./Request');
+
+const bodyRequired = ['applicantId', 'comments', 'favorite'];
+const bodyValidations = {
+  'applicantId': ['required', 'naturalNonZero'],
+  'comments': ['string', 'maxLength:255'],
+  'favorite': ['natural']
```

Comment:
boolean

Reply:
I think it would give more flexibility to have it be integers so that they could do a star rating if they wanted to